traits/trait_types.pyi

```diff
@@ -8,13 +8,16 @@
 #
 # Thanks for using Enthought open source!
 
+from __future__ import annotations
```

Comment:
POssibly we don't need this in an annotations file. I'll see if I can remove it.